apps/vue-app/src/app/modules/categories/components/CategoryForm.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+<div class="modal fade" id="createCategoryModal" tabindex="-1" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered">
+        <div class="modal-content">
+            <div class="modal-header text-center">
+                <h5 class="modal-title">{{ action }} Category</h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"
+                    @click="reset()"></button>
+            </div>
+            <div class="modal-body">
+                <form @submit.prevent="submit">
+                    <div class="form-group pb-3">
+                        <label>Name</label>
+                        <input type="text"
+                            :class="v$?.categorySelected?.name?.$error === true ? 'form-control is-invalid' : 'form-control'"
+                            v-model="categorySelected.name" />
```

Comment:
When you are editing a category the table also updates that values, it happens because you are using `categorySelected.name` in the `v-model`, to avoid this you can use your `category` from line 52

Mutating a prop is usually not recommended